continuous-delivery/publish_to_prod_pypi.yml

```diff
@@ -1,5 +1,5 @@
 version: 0.2
-# this image assumes Ubuntu 14.04 base image
```

Comment:
did you update the all the codebuild jobs in the release pipeline to use the newer docker images?

Reply:
yup. All relevant codebuild environments have been changed to aws-crt-ubuntu-20-x64:latest